docs/Troubleshooting.md

```diff
@@ -95,3 +95,20 @@ If you run into issues where running `react-native init` hangs in your system, t
 ```
 react-native init --verbose
 ```
+
+## Unable to execute JS call: __fbBatchedBridge is undefined
+Refer to issue [#6282](https://github.com/facebook/react-native/issues/6282)
+
+In `node_modules/react-native/packager/react-native-xcode.sh` comment out the following lines:
+
+```
+if [[ "$PLATFORM_NAME" = "iphonesimulator" ]]; then
+  echo "Skipping bundling for Simulator platform"
+  exit 0;
+fi
+```
+## Cannot refresh page in simulator:
```

Comment:
So it looks like most of these are already tracked in GitHub. I recently went through this Troubleshooting page and removed anything already tracked elsewhere. I don't think Troubleshooting is meant to be an exhaustive page that list every single issue. I'd actually like to get it to a point where we don't have a troubleshooting page and instead link to GitHub where people can find the most up to date discussion on any particular issue.